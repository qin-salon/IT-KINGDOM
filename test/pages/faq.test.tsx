import FaqPage from "src/pages/faq";
import { render } from "test/test-utils";

describe("FaqPage", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<FaqPage />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
