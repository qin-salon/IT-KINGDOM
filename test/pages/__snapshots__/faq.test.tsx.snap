// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FaqPage matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="container"
  >
    <h1>
      <a
        href="/"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27180%27%20height=%2738%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="IT KINGDOM logo"
            data-nimg="intrinsic"
            decoding="async"
            src="/_next/image?url=%2Fimg%2Flogo.png&w=384&q=75"
            srcset="/_next/image?url=%2Fimg%2Flogo.png&w=256&q=75 1x, /_next/image?url=%2Fimg%2Flogo.png&w=384&q=75 2x"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
        </span>
      </a>
    </h1>
    <nav
      class="sansSerif nav"
    >
      <ul>
        <li>
          <a
            href="/faq"
          >
            よくある質問
          </a>
        </li>
        <li>
          <a
            href="https://qin.salon/signup"
            rel="noopener noreferrer"
            target="_blank"
          >
            入国手続
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main
    class="container"
  >
    <h1
      class="title"
    >
      よくある質問
    </h1>
    <ul>
      <li
        class="category"
      >
        <h2>
          サロンの内容について
        </h2>
        <ul>
          <li
            class="qa"
          >
            <h3>
              Q どのような方が対象ですか？
            </h3>
            <p>
              IT業界に興味のある方やプログラミングに興味のある方にオススメしております。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q プログラミング初心者ですが参加可能ですか？
            </h3>
            <p>
              もちろん大歓迎です。初心者から経験者まで誰でもご参加いただけます。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q 参加したらプログラミングができるようになりますか？
            </h3>
            <p>
              参加しただけでプログラミングができるようになるものではございません。自主学習と併用してご活用いただくことをオススメしています。当サロンでは「一人だとモチベーションが維持できない」「詰まったときに答えが見つからない」といったプログラミング学習における課題を解決できるための仕組みを用意しております。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q なぜしまぶーさん以外に運営がいるのですか？
            </h3>
            <p>
              サロンの人数が多くなると考え、そうなった場合に1人で運営を行うことが不可能だと判断したためです。また当サロンは実際にプロダクトを作るサロンで、私一人の開発では限界があります。私以上にプログラミング能力がある方を運営に招くことで、プロダクトの開発速度とクオリティをアップさせたいと考えました。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q オフラインのイベントもありますか？
            </h3>
            <p>
              新型コロナウイルス感染症が収まり次第、オフラインイベントも行う予定です。東京に偏りすぎないように地方でもイベントを行いたいと考えております。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q マンガ『キングダム』と関係ありますか？
            </h3>
            <p>
              IT KINGDOMは原泰久先生の作品『キングダム』とは一切関係がない非公式のサービスです。オマージュとして設定を一部拝借しております。
            </p>
          </li>
        </ul>
      </li>
      <li
        class="category"
      >
        <h2>
          お支払いやアカウントについて
        </h2>
        <ul>
          <li
            class="qa"
          >
            <h3>
              Q どのような支払い方法がありますか？
            </h3>
            <p>
              クレジットカードなど、Stripeが指定するお支払い方法をご利用いただけます。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q 入会後、初回料金はいつ決済されますか？
            </h3>
            <p>
              入会手続き時に決済が行われます。その後、入会日から1ヶ月ごとに決済が行われます。いつ入会しても同じ条件で、入会日によって金額的に損する仕組みにはなっておりません。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q 領収書はどこで発行できますか？
            </h3>
            <p>
              毎月の決済のタイミングで送られるメールから領収書をダウンロードできます。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q クレジットカード情報、登録メールアドレス、その他の情報を変更したいです
            </h3>
            <p>
              ログイン後のマイページから変更可能です。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q 退会方法を教えてください
            </h3>
            <p>
              ログイン後のマイページより退会手続きが行えます。
            </p>
          </li>
          <li
            class="qa"
          >
            <h3>
              Q キャンセルしたいです
            </h3>
            <p>
              申し訳ございませんが原則キャンセルは承っておりません。参加後に満足いただけなかった場合は退会をお願いしております。なお退会における返金は受け付けておりませんので、参加前に十分なご検討をお願いいたします。
            </p>
          </li>
        </ul>
      </li>
    </ul>
    <p
      class="note"
    >
      その他ご不明点がございましたら、
      <a
        href="https://docs.google.com/forms/d/e/1FAIpQLSf7lYhGHmnl2KTPXsArv5KCoZvlwX7pMF7EZrDXSgWQWnJDhw/viewform"
        rel="noopener noreferrer"
        target="_blank"
      >
        お問い合わせ
      </a>
      よりご連絡ください。
    </p>
  </main>
  <footer
    class="sansSerif container"
  >
    <ul
      class="links"
    >
      <li
        class="link"
      >
        <a
          href="/support/terms"
        >
          利用規約
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="/support/privacy-policy"
        >
          プライバシーポリシー
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="/support/commerce"
        >
          特定商取引法に基づく表記
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="https://docs.google.com/forms/d/e/1FAIpQLSf7lYhGHmnl2KTPXsArv5KCoZvlwX7pMF7EZrDXSgWQWnJDhw/viewform"
          rel="noopener noreferrer"
          target="_blank"
        >
          お問い合わせ
        </a>
      </li>
    </ul>
    <small
      class="copyright"
      lang="en"
    >
      @ 2020 IT KINGDOM
    </small>
  </footer>
</DocumentFragment>
`;
